libopae/src/common.c

```diff
@@ -184,3 +184,29 @@ uint64_t wsid_gen(void)
 	id |= ((unsigned long) getpid() % 16777216) << 24;
 	return id;
 }
+
+/**
+ * @brief
+ *
+ * @param guidh
+ * @param guidl
+ * @param guid
+ */
+void __FIXME_MAKE_VISIBLE__ fpga_guid_to_fpga(uint64_t guidh, uint64_t guidl, uint8_t *guid)
```

Comment:
Hey Ananda, sorry for the trouble, but I had a recent pull request that removed this function (it was unused in the API).  Do you think it makes sense to move it into fpgaconf, since that's the only entity that uses it?